CMakeLists.txt

```diff
@@ -112,9 +121,11 @@ ELSE()
     SET_TARGET_PROPERTIES(test_cpu PROPERTIES COMPILE_FLAGS "${CMAKE_CXX_FLAGS} --std=c++11")
 
     INSTALL(TARGETS warpctc
-            RUNTIME DESTINATION "${CMAKE_BINARY_DIR}"
-            LIBRARY DESTINATION "${CMAKE_BINARY_DIR}"
-            ARCHIVE DESTINATION "${CMAKE_BINARY_DIR}")
+            RUNTIME DESTINATION "${CMAKE_INSTALL_PREFIX}/bin"
+            LIBRARY DESTINATION "${CMAKE_INSTALL_PREFIX}/lib"
+            ARCHIVE DESTINATION "${CMAKE_INSTALL_PREFIX}/lib")
+
+    INSTALL(DIRECTORY include/ DESTINATION "${CMAKE_INSTALL_PREFIX}/include")
```

Comment:
Is there a reason to install the entire `include/` directory? I believe users should only need the `include/ctc.h` file.